opamp-client/src/http/sender.rs

```diff
@@ -14,21 +16,24 @@ where
 {
     compressor: Compressor,
     client: C,
+    instance_uid: InstanceUid,
 }
 
 impl<C> HttpSender<C>
 where
     C: HttpClient,
 {
     // Initializes a new instance of HttpSender with the provided HTTP client.
-    pub(super) fn new(client: C) -> Self {
+    pub(super) fn new(client: C, instance_uid: InstanceUid) -> Self {
         Self {
             compressor: Compressor::Plain,
             client,
+            instance_uid,
         }
     }
 
     // Sends an AgentToServer message using the HttpSender and returns an optional ServerToAgent message as a result.
+    #[instrument(name = "opamp::send",fields(instance_uid = self.instance_uid.to_string(),sequence_number = msg.sequence_num), skip(msg,self))]
```

Comment:
there is the skip_all if you do not want to list things

Reply:
Added 👍  Thanks!